File: src/main/java/nuclearcoder/discordbot/command/misc/CmdKiss.java
package nuclearcoder.discordbot.command.misc;

import nuclearcoder.discordbot.BotUtil;
import nuclearcoder.discordbot.NuclearBot;
import nuclearcoder.discordbot.command.Command;
import sx.blah.discord.handle.obj.IChannel;
import sx.blah.discord.handle.obj.IMessage;
import sx.blah.discord.handle.obj.IUser;

import java.util.Arrays;

public class CmdKiss implements Command {

    @Override public void execute(NuclearBot bot, IMessage message, String command, String[] args)
    {
        if (args.length >= 2)
        {
            final IChannel channel = message.getChannel();
            final IUser user = message.getAuthor();

            final String target = String.join(" ", Arrays.copyOfRange(args, 1, args.length)).trim();

            BotUtil.sendMessage(channel,
                    user.mention() + " kisses " + target + "  ( \u02D8 \u00B3\u02D8) ~~");
        }
    }

}

File: src/main/java/nukebot/command/misc/CmdKiss.kt
package nukebot.command.misc

import nukebot.NuclearBot
import nukebot.command.Command
import nukebot.rSendMessage
import sx.blah.discord.handle.obj.IMessage

class CmdKiss : Command {

    override fun execute(bot: NuclearBot, message: IMessage, command: String, args: Array<String>) {
        if (args.size >= 2) {
            val channel = message.channel
            val user = message.author

            val target = args.sliceArray(1..args.lastIndex).joinToString(" ").trim()

            channel.rSendMessage(user.mention() + " kisses " + target + "  ( \u02D8 \u00B3\u02D8) ~~")
        }
    }

}
